nova-components/WaitingListsManager/src/Http/WaitingListAccountResource.php

```diff
@@ -16,9 +16,9 @@ public function toArray($request)
             'id' => $this->id,
             'pivot_id' => $this->pivot->id,
             'name' => $this->name,
-            'atcHourCheck' => $this->pivot->atcHourCheck,
+            'atcHourCheck' => $this->pivot->eligibility_summary['base_controlling_hours'] ?? false,
```

Comment:
Maybe use a graceful array accessor here to prevent any exceptions if the index doesn't exist?
`Arr::get($this->pivot->eligibility_summary, 'base_controlling_hours', false)`

Reply:
nice one!